install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -151,13 +151,14 @@ data:
           {{ "[[ else -]]" }}
           runAsUser: 1337
           {{ "[[- end ]]" }}
+        {{ "[[ if (isset .ObjectMeta.Annotations `sidecar.istio.io/proxyCPU`) -]]" }}
```

Comment:
this change appears a little suspicious.  Would like @ayj to take a look before approval.